docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -99,7 +109,7 @@ This method of provisioning the cluster would act as a single control point for
 We are fully aware that in order to exploit the potential of ClusterClass and managed topologies, the following class of problems still needs to be addressed:
 - **Lifecycle of the ClusterClass**: Introduce mechanisms for allowing mutation of a ClusterClass, and the continuous reconciliation of the Cluster managed resources.
```

Comment:
IMO this line should be changed or removed as this amendment specifically introduces mutable parts to a ClusterClass.

Reply:
Yeah agree. My problem is that ClusterClass is already mutable even without introducing patches (you can change e.g. templates referenced in the ClusterClass or metadata). 

So the operational benefits (like automatic template rotation) are not patch-specific at all. I'll try to sort this out and bring it in a consistent state.

Reply:
I tried to clarify it be adding user stories for mutability (independent of patches) and updating the update cluster behavior section.